settings.adoc

```diff
@@ -5,8 +5,8 @@
 :product-short: Zimbra
```

Comment:
Please exclude the settings file from this PR. It will be sufficient to modify a local version of the file to build the Japanese 8.8.12 output.